client/dashboard/components/action-list/index.tsx

```diff
@@ -0,0 +1,45 @@
+import {
+	__experimentalVStack as VStack,
+	__experimentalText as Text,
+	Card,
+} from '@wordpress/components';
+import { forwardRef } from 'react';
+import ActionItem from './action-item';
+import type { ActionListProps } from './types';
+import './style.scss';
+
+function UnforwardedActionList(
+	{ title, description, children }: ActionListProps,
+	ref: React.ForwardedRef< HTMLAnchorElement | HTMLButtonElement >
+) {
+	return (
+		<Card className="action-list" ref={ ref }>
+			{ ( title || description ) && (
```

Comment:
Should we render `<CardBody />` to surround the content?

Reply:
Added.